udaykalra/walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -40,11 +61,20 @@
     availableRanges.add(fullDay);
 
     // Obtain data from the request.
-    Collection<String> attendees = request.getAttendees();
+    Collection<String> attendees = new HashSet<>();
+    Collection<String> mandatoryAttendees = request.getAttendees();
+    Collection<String> optionalAttendees = request.getOptionalAttendees();
     long requestDuration = request.getDuration();
 
     Collection<TimeRange> busyRanges = new HashSet<>();
 
+    attendees.addAll(mandatoryAttendees);
+
+    // Include Optional Attendees if indicated.
+    if (withOptional) {
+      attendees.addAll(optionalAttendees);
+    }
```

Comment:
I would be clearer to perform this closer to the declaration (or else move the declaration just above this block). The idea is that reducing the spread of related code makes it easier for readers to follow flow.

Reply:
Done